healthcheck/database.go

```diff
@@ -0,0 +1,38 @@
+package healthcheck
+
+import "database/sql"
+
+// HealthDatabase allows to execute a simple one-row-result SQL query
+type HealthDatabase interface {
+	QueryRow(query string, args ...interface{}) *sql.Row
+}
+
+type databaseChecker struct {
+	alias string
+	dbase HealthDatabase
+}
+
+// newDatabaseChecker creates a database health checker
+func newDatabaseChecker(alias string, dbase HealthDatabase) BasicChecker {
+	return &databaseChecker{
+		alias: alias,
+		dbase: dbase,
+	}
+}
+
+func (dbc *databaseChecker) CheckStatus() healthStatus {
+	var value int
+	var res = healthStatus{}
+
+	row := dbc.dbase.QueryRow("select 1")
```

Comment:
you could use Ping instead of QueryRow if it's just for checking the connection

Reply:
Thanks... You already suggested Ping in another PR but I forgot it... It's easier to mock and this way, we can reach a coverage of 100% !